CondTools/Ecal/src/EcalTPGOddWeightIdMapHandler.cc

```diff
@@ -213,44 +212,44 @@ void popcon::EcalTPGOddWeightIdMapHandler::getNewObjects() {
 
               writeFile("last_tpg_OddweightIdMap_settings.txt");
 
-              std::cout
-                  << " even if the tag/version is not the same, the weightIdMap id is the same -> no transfer needed "
-                  << std::endl;
+              edm::LogVerbatim("EcalTPGOddWeightIdMapHandler")
+                  << " even if the tag/version is not the same, the weightIdMap id is the same -> no transfer needed ";
             }
 
           } catch (std::exception& e) {
-            std::cout << "ERROR: THIS CONFIG DOES NOT EXIST: tag=" << the_config_tag
-                      << " version=" << the_config_version << std::endl;
-            std::cout << e.what() << std::endl;
+            throw cms::Exception("FileReadError") << "ERROR: THIS CONFIG DOES NOT EXIST: tag=" << the_config_tag
+                                                  << " version=" << the_config_version << "\n"
+                                                  << e.what();
             m_i_run_number = irun;
           }
```

Comment:
I think you should not catch the exception, but let the framework do it, please see 7.11 in https://cms-sw.github.io/cms_coding_rules.html